shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/MainScreen.kt

```diff
@@ -59,30 +60,33 @@ fun MainScreen(service: ConferenceService) {
                 }, onRejectPrivacy = {}
             )
         } else {
+            val agendaScrollState = rememberLazyListState()
+            val speakersScrollState = rememberLazyListState()
+
             TabsView(
                 controller,
                 TabItem("menu", Res.drawable.menu, Res.drawable.menu_active) {
                     MenuScreen(controller = it)
                 },
                 TabItem("agenda", Res.drawable.time, Res.drawable.time_active) {
-                    AgendaView(agenda, it)
+                    AgendaScreen(agenda, agendaScrollState, it)
                 },
                 TabItem(
                     "speakers",
                     Res.drawable.speakers,
                     Res.drawable.speakers_active
                 ) {
-                    SpeakersView(controller = it, speakers = speakers.all)
+                    SpeakersScreen(controller = it, scrollState = speakersScrollState, speakers = speakers.all)
                 },
                 TabItem(
-                    "Bookmarks",
+                    "bookmarks",
                     Res.drawable.mytalks,
                     Res.drawable.mytalks_active
```

Comment:
the same here: icons and texts should be resource ids

Reply:
let me check